lerobot/common/robot_devices/control_utils.py

```diff
@@ -289,7 +289,10 @@ def control_loop(
             break
 
 
-def reset_environment(robot, events, reset_time_s, fps):
+def reset_environment(robot, events, reset_time_s, fps, policy):
+    if policy is not None:
+        policy._action_queue.clear()
```

Comment:
- `reset_environment` is a function that allows to teleop the robot arm to place it to a reset position. I think there is a better place to move your logic: `control_loop` 
- Why not use `policy.reset()`?

Reply:
Hey, only `Act` and `Pi0` have the `_action_queue` attribute.
We have two options: either check if the _action_queue attribute exists, or implement this attribute in all the policies.
Alternatively, we could use `policy.reset()`, which I think is the cleanest solution since it avoids mutating a private attribute from outside its class.